src/Collections/TerminusCollection.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace Pantheon\Terminus\Collections;
+
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
+use Pantheon\Terminus\Request\RequestAwareTrait;
+use Terminus\Exceptions\TerminusException;
+use Pantheon\Terminus\Request;
+
+abstract class TerminusCollection implements Request\RequestAwareInterface, ContainerAwareInterface
```

Comment:
Line 11 should use `RequestAwareTrait` and a use statement added, or line 13 should use `Request\RequestAwareTrait` and the use statement for `RequestAwareTrait` removed. I prefer the former.